package.json

```diff
@@ -52,30 +52,34 @@
     "url": "https://github.com/alex3165/react-mapbox-gl/issues"
   },
   "homepage": "https://github.com/alex3165/react-mapbox-gl#readme",
-  "dependencies": {
+  "dependencies": {},
+  "peerDependencies": {
     "deep-equal": "^1.0.1",
     "mapbox-gl": "^0.35.0",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
     "reduce-object": "^0.1.3",
     "supercluster": "^2.3.0"
   },
-  "peerDependencies": {
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2"
-  },
   "devDependencies": {
     "@types/core-js": "^0.9.41",
+    "@types/deep-equal": "^1.0.0",
     "@types/enzyme": "^2.7.7",
     "@types/geojson": "^1.0.1",
     "@types/jest": "^19.2.2",
+    "@types/mapbox-gl": "^0.30.0",
```

Comment:
I think it is a good idea, I tried something like :
```
declare module 'mapbox-gl' {
  export class Evented extends mapboxgl.Evented {
    on(type: string, listener: Function): this;
    on(type: string, layerId: string, listener: Function): this;
  }

  export class Map extends mapboxgl.Map {
    setFilter(layer: string, filter?: any[]): this;
  }
}

```

It could work but I get the error : `Duplicate identifier 'Map'` or `Duplicate identifier 'Evented'`